docs/http_api.rst

```diff
@@ -378,7 +424,14 @@ to belong to exactly one organization, but an organization can have many teams.
        HTTP/1.1 200 OK
        Content-Type: application/json
 
-       {"title":"Brotherhood Without Banners","id":4,"teams":[],"url":"https://example.org/organizations/4","users":[]}
+       {
+        "title":"Brotherhood Without Banners",
+        "id":4,
+        "teams":[],
+        "url":"https://example.org/organizations/4",
```

Comment:
We don't always need the trailing slash, although I think we can always have a trailing slash. Let me check it, and change it to be all uniform.